enterprise/frontend/src/metabase-enterprise/transforms/components/QueryEditor/NativeQuerySidebar/NativeQuerySidebar.tsx

```diff
@@ -0,0 +1,80 @@
+import { ResizableBox } from "react-resizable";
+import { useWindowSize } from "react-use";
+import { t } from "ttag";
+
+import { ControlledNotebookNativePreview } from "metabase/querying/notebook/components/NotebookNativePreview/NotebookNativePreview";
+import { ActionIcon, Icon, Tooltip } from "metabase/ui";
+import type Question from "metabase-lib/v1/Question";
+
+import { ResizeHandle } from "../ResizeHandle";
+
+import S from "./NativeQuerySidebar.module.css";
+
+export function NativeQuerySidebarToggle({
+  canConvertToNative,
+  isShowingNativeQueryPreview,
+  onToggleNativeQueryPreview,
+}: {
+  canConvertToNative?: boolean;
+  isShowingNativeQueryPreview: boolean;
+  onToggleNativeQueryPreview: () => void;
+}) {
+  if (!canConvertToNative) {
```

Comment:
I see you only pass `isNative` for this prop. Perhaps just call it here the same?